src/build.ts

```diff
@@ -32,7 +33,9 @@ function getGenerateCommand(buildInfo: BuildInfo, ctx: ExtensionContext) {
     result.args.push(getTargetCommand(buildInfo.target));
   }
 
-  result.args.push('-o', buildInfo.projectName, getActiveEditorPath());
+  // todo: it is possible to get an activated window identifier
+  // const grammarFile = getActiveEditorPath();
+  result.args.push('-o', buildInfo.projectName, grammarFile);
```

Comment:
Because I added output, it is possible that getActiveEditorPath obtained output, which caused other issues, so I passed the syntax file through